inventory/ipv4_addresses/inventory_ipv4_addresses.cf

```diff
@@ -0,0 +1,24 @@
+bundle agent inventory_ipv4_addresses
+{
+  vars:
+    "filter_reg"
+      string => "127\.0\.0\.1",
```

Comment:
I would say that loopback addresses that are not 127.0.0.1 would be interesting, its definately not common. But I can change it to 127\.\d{0,3}\.\d{0,3}\.\d{0,3} if you prefer.